physics/gwdps.f

```diff
@@ -308,10 +313,16 @@ subroutine gwdps_run(                                             &
       real(kind=kind_phys), intent(inout) :: ELVMAX(:)
       real(kind=kind_phys), intent(in) ::                               &
      &                     THETA(:), SIGMA(:), GAMMA(:)
-      real(kind=kind_phys), intent(out) :: DUSFC(:), DVSFC(:),          &
+      real(kind=kind_phys), intent(inout) :: DUSFC(:), DVSFC(:),        &
      &                     RDXZB(:)
+      real(kind=kind_phys), intent(inout) :: dtaux2d_ms(IM,KM),         &
```

Comment:
These are defined intent(inout) in the meta file?

Reply:
Changed to intent(inout) in the meta file.